api/views.py

```diff
@@ -430,16 +439,16 @@ def all_activity_over_time(self, request, format=None):
             )
         ).filter(instructors=0).values_list('slug', flat=True)
 
-        return Response({
+        return {
             'start': start,
             'end': end,
             'workshops': {
                 'SWC': swc_workshops.count(),
                 'DC': dc_workshops.count(),
-                'SWC,DC': swc_dc_workshops,
+                'SWC_or_DC': swc_dc_workshops,
```

Comment:
It was used here: https://github.com/swcarpentry/amy/pull/1060/commits/6239fa34d6fb0c4798a5ac2831a5040623493ffe#diff-345991cbd83965baed2fa14d0e2f37ceL106